test/check-application

```diff
@@ -672,6 +672,54 @@ class TestNavigator(testlib.MachineCase):
         self.assertEqual(m.execute("ls -l /home/admin/newdir/file1")[:10], "-rwxrwxrwx")
         self.assertEqual(m.execute("ls -l /home/admin/newdir/file2")[:10], "-rwxrwxrwx")
 
+    def testKeyboard(self):
+        b = self.browser
+        m = self.machine
+
+        self.restore_dir("/home/admin")
+
+        self.enter_navigator()
+
+        # FIXME: Use a loop here? (running m.execute inside for loop doesn't work)
+        m.execute("touch /home/admin/file1 /home/admin/file2 /home/admin/file3 /home/admin/file4 /home/admin/file5 /home/admin/file6 /home/admin/file7")
+        
+        # Test left/right
+        b.click("[data-item='file1']")
+        self.assertIn("pf-m-selected", b.attr("[data-item='file1']", "class"))
+        b.click("[data-item='file2']") # b.key_press(chr(39), use_ord=True)
```

Comment:
The issue is here that if you add a `console.log` to `onKeyboardNav` no event shows up when you try to use `b.key_press`. So for some reason the event is not initialized or focussed.